tests/cases/resource_test.py

```diff
@@ -455,11 +455,21 @@ def testGetResourceByPath(self):
                 str(resp.json['_id']), str(item['_id']))
 
         # test bogus path
+        # test is set to false, response code should be 400
```

Comment:
Can you add another test, replicating the old behavior?  i.e., Don't pass a value for `test` and ensure you get the 400 exception.